src/components/auth/Form.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import { signup } from '../../state/auth/authActionCreator';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['signup' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5668611)